procurement_purchase_groupby_analytic/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Procurement Purchase Group by Analytic
+===========
+
+This module group purchase order from sale order procurement by analytic.
+When purchase order is created from procurement order it sets analytic
+on purchase order line.
```

Comment:
All purchase orders created from a procurement order linked to a sale order with analytic account